rotkehlchen/db/history_events.py

```diff
@@ -174,13 +256,29 @@ def edit_history_event(self, write_cursor: 'DBCursor', event: HistoryBaseEntry)
             else:  # all other data
                 write_cursor.execute(f'{updatestr} WHERE identifier=?', (*bindings, event.identifier))  # noqa: E501
 
-        # Also mark it as customized
         write_cursor.execute(
             'INSERT OR IGNORE INTO history_events_mappings(parent_identifier, name, value) '
             'VALUES(?, ?, ?)',
             (event.identifier, HISTORY_MAPPING_KEY_STATE, HISTORY_MAPPING_STATE_CUSTOMIZED),
         )
 
+        # Track modification only if balance-affecting fields changed. cannot be None here
+        if old_data == (
+            event.timestamp,
+            event.asset.identifier,
+            str(event.amount),
+            event.event_type.serialize(),
+            event.event_subtype.serialize(),
+            event.location_label,
+        ):
+            return
+
+        old_ts, _, _, _, _, _ = old_data
+        self._mark_events_modified(
+            write_cursor=write_cursor,
+            timestamp=TimestampMS(min(old_ts, event.timestamp)),
+        )
```

Comment:
Why unpack all values here?

```suggestion
        self._mark_events_modified(
            write_cursor=write_cursor,
            timestamp=TimestampMS(min(old_data[0], event.timestamp)),
        )
```